docs/en/engines/database_engines/mysql.md

```diff
@@ -19,7 +19,7 @@ You cannot perform the following queries:
 
 ``` sql
 CREATE DATABASE [IF NOT EXISTS] db_name [ON CLUSTER cluster]
-ENGINE = MySQL('host:port', 'database', 'user', 'password')
+ENGINE = MySQL('host:port', ['database' | database], 'user', 'password')
```

Comment:
Actually now I can't understand why we allow both variants here and have test `test_bad_arguments_for_mysql_database_engine` later.

There is different behavior in Storage and TableFunction?
```sql
CREATE TABLE default.t1_remote_mysql AS mysql('mysql1:3306', 'test', 't1_mysql_local', root, 'clickhouse')
```